### DISCUSSION TITLE
Refresh database default tables

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#14130] [How to]
I have database from older version of YF and new files belonging to v.6. Now, without migrating I would like to upgrade older database so all default fields matching to new set of files. 
Is any script out there? 
Exact version of previous yetiforce is unknown, But surely it was 5.x

**Comment by mariuszkrzaczkowski** [comment]
```sql
SELECT * FROM vtiger_version
```

**Reply by TonyM15** [reply]
yes sure... the only thing is, there was possibly little mess while migration, so version might not be aligned with database structure and default fields values.  It could be mix of alter ,create, update statements, what actually would be cool for the migration script. This approach could save those versions checks.

**Reply by rskrzypczak** [reply]
There is no such script, and you shouldn't even try it.
Updates are based on files of a specific version.
The only thing you can do is upload files from the version adequate to the database and update.
